ambry-quota/src/main/java/com/github/ambry/quota/capacityunit/AmbryQuotaManager.java

```diff
@@ -0,0 +1,143 @@
+/*
+ * Copyright 2021 LinkedIn Corp. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ */
+package com.github.ambry.quota.capacityunit;
```

Comment:
Did you intend to put this class in this package?  Doesn't seem to belong.

Reply:
fixed.